src/common/CMakeLists.txt

```diff
@@ -33,6 +33,7 @@ surge_add_lib_subdirectory(sst/sst-cpputils)
 surge_add_lib_subdirectory(sst/sst-plugininfra)
 surge_add_lib_subdirectory(sst/sst-filters)
 surge_add_lib_subdirectory(sst/sst-waveshapers)
+surge_add_lib_subdirectory(sst/sst-jucegui)
```

Comment:
This inclusion requires a `if (NOT SURGE_SKIP_JUCE_FOR_RACK)` / `endif()` block surrounding it. Thanks!

Reply:
i moved it down into the existing block